lib/ansible/modules/crypto/openssl_certificate_info.py

```diff
@@ -238,10 +238,45 @@
                  or not.
     returned: success
     type: dict
+subject_key_identifier:
+    description:
+        - The CSR's subject key identifier.
+        - The identifier is returned in hexadecial, with C(:) used to separate bytes.
+        - Is C(none) if the C(SubjectKeyIdentifier) extension is not present.
+    returned: success and if the pyOpenSSL backend is I(not) used
+    type: str
+    sample: '00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff:00:11:22:33'
+    version_added: "2.9"
+authority_key_identifier:
+    description:
+        - The CSR's authority key identifier.
```

Comment:
```suggestion
        - The certificate's authority key identifier.
```

Reply:
Thanks for spotting this (and the related) copy'n'paste errors and typos!